<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reghaabat</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Tahoma;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="container" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDeliver"/>
   <addaction name="actionNewUser"/>
   <addaction name="actionEditUser"/>
   <addaction name="separator"/>
   <addaction name="actionUserManagement"/>
   <addaction name="actionObjectManagement"/>
   <addaction name="actionMatchManagement"/>
   <addaction name="actionSetScores"/>
   <addaction name="actionPayment"/>
   <addaction name="separator"/>
   <addaction name="actionWeb"/>
   <addaction name="actionChangePermissions"/>
   <addaction name="actionOptions"/>
   <addaction name="separator"/>
   <addaction name="actionChangePassword"/>
   <addaction name="actionLogin"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionWeb">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/web.png</normaloff>:/images/web.png</iconset>
   </property>
   <property name="text">
    <string>Web</string>
   </property>
   <property name="toolTip">
    <string>Web Connection</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/login.png</normaloff>:/images/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionDeliver">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/deliver.png</normaloff>:/images/deliver.png</iconset>
   </property>
   <property name="text">
    <string>Deliver</string>
   </property>
   <property name="toolTip">
    <string>Deliver</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/password.png</normaloff>:/images/password.png</iconset>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionChangePermissions">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/permissions.png</normaloff>:/images/permissions.png</iconset>
   </property>
   <property name="text">
    <string>Change Permissions</string>
   </property>
  </action>
  <action name="actionSetScores">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/setscores.png</normaloff>:/images/setscores.png</iconset>
   </property>
   <property name="text">
    <string>Set Scores</string>
   </property>
   <property name="toolTip">
    <string>Set Scores</string>
   </property>
  </action>
  <action name="actionUserManagement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
   </property>
   <property name="text">
    <string>User Management</string>
   </property>
  </action>
  <action name="actionMatchManagement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/match.png</normaloff>:/images/match.png</iconset>
   </property>
   <property name="text">
    <string>Match Management</string>
   </property>
   <property name="toolTip">
    <string>Match Management</string>
   </property>
  </action>
  <action name="actionPayment">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/payment.png</normaloff>:/images/payment.png</iconset>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </action>
  <action name="actionNewUser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/add_user.png</normaloff>:/images/add_user.png</iconset>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionEditUser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/edit_user.png</normaloff>:/images/edit_user.png</iconset>
   </property>
   <property name="text">
    <string>Edit User</string>
   </property>
  </action>
  <action name="actionObjectManagement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/object.png</normaloff>:/images/object.png</iconset>
   </property>
   <property name="text">
    <string>Object Management</string>
   </property>
   <property name="toolTip">
    <string>Object Management</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/reghaabat.ico</normaloff>:/images/reghaabat.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
